### DISCUSSION TITLE
5.x开启controller模式broker启动失败，NullPointerException

### DISCUSSION THREAD

**Discussion by xiaokaijunjun** [apache/rocketmq#7333] [Q&A]
版本：5.1.2
系统：Linux
JDK：1.8.0_381

只要broker设置enableControllerMode=true 开启controller，broker启动报错：java.lang.NullPointerException，broker配置单纯把这行去掉broker就能启动，请问是什么原因呢

**Comment by joeCarf** [comment]
麻烦贴一下详细的报错栈，感谢～

**Comment by RongtongJin** [comment]
https://github.com/apache/rocketmq/discussions/6928
5.1.2版本存在问题，可以升到5.1.3
